pkg/services/featuremgmt/registry.go

```diff
@@ -1,3 +1,8 @@
+// To change feature flags, edit:
```

Comment:
Also paste the instructions at the top of the registry file, for posterity. That way, future developers who touch this file have a hint to look elsewhere.